ivy/data_classes/container/base.py

```diff
@@ -4159,7 +4159,9 @@ def __getstate__(self):
         state_dict["_config_in"] = config_in
         config = copy.copy(state_dict["_config"])
         config["ivyh"] = (
-            config["ivyh"].current_backend_str() if config["ivyh"] is not None else None
+            config["ivyh"].current_backend_str()
+            if getattr(config, "ivyh", None) is not None
+            else None
```

Comment:
Can we revert unrelated changes?

Reply:
This is because of some corner case where `config["ivyh"]` is undefined as the key `ivyh` isn't present there, might be a mp thing, but not sure, the replaced code handles this corner case too.